### DISCUSSION TITLE
Instalation

### DISCUSSION THREAD

**Discussion by flvoicu** [AndresGarciaSobrado91/MSI-Modern15-Hackintosh#13] [Q&A] [answered]
Hello,  I don't really know how to start 😅, so here are some question:
Do I need a MacOS machine?  
Could you make a breve explanation on how to install it, or maybe recomend a tutorial? 
Thanks

**Comment by AndresGarciaSobrado91** [comment, marked as answer]
Hi! I can recommend you: https://dortania.github.io/OpenCore-Install-Guide/installer-guide/

Sections you must follow:

- USB Creation

- Installation

So, in the USB you have to set the macOS installer and this EFI folder.

**Reply by flvoicu** [reply]
First of all thanks for the reply,  but I have one more question, Do I need to instal a full macos in the usb (using a macbook)?  or mayme I could make an online install if I use a dongle (usb to ethernet)?

**Reply by AndresGarciaSobrado91** [reply]
I recommend you to get the full offline macOS iso (11.2.3 or older Big Sur installer) into your USB from a running macOS. If you don't have a running macOS, you could try with the eth dongle

**Reply by flvoicu** [reply]
oke, muchas gracias

**Reply by sysoev-dev** [reply]
Qué adaptador de Internet usaste?

**Reply by flvoicu** [reply]
Uno de ethernet a usb de amazon, marca tp-link

**Comment by flvoicu** [comment]
Buenas, al final conseguí instalarlo con el adaptador, pero no consigo que funcione el brillo,me podrias dar alguna sugerencia?

**Reply by flvoicu** [reply]
Ah y una cosa mas, si el pc "sleeps" se queda con la pantalla en gris y tengo que hacer un hard reset.

**Reply by AndresGarciaSobrado91** [reply]
estas usando todos los archivos de mi repo? es tu modelo exactamente igual al mio? suena a que no tienes un mapeo usb correcto

**Reply by flvoicu** [reply]
Hola, si es tu EFI y mi modelo es el 10ras-098xes.
Podria ser debido a que desbloquee el CFG despues de instalar el hakintosh?

**Reply by AndresGarciaSobrado91** [reply]
Ok, es otro modelo (el mio 088XES), aparentemente tu modelo difiere en algunas cuestiones como la falta de un puerto USB type-C. Tendrías que eliminar el USBMap.kext y crear uno nuevo para tu modelo en particular

**Reply by flvoicu** [reply]
Perdon 3s el 89*, a la izquierda tengo un usb , un usb c, el hdmi y un jack de 3.5; a la derecha 2 usb y el lector de tarjetas sd

**Reply by flvoicu** [reply]
Ademas al iniciarlo me dan unos errores como el boot-signature y boot-image-key

**Reply by flvoicu** [reply]
Creo que la diferencia es que el mio tiene un mx330, podria ser ese el error?

**Reply by flvoicu** [reply]
Vale, era culpa de la grafica, la he desactivado y todo va perfecto. Muchas gracias por la repo.
